tests/Yaapii.Atoms.Tests/Scalar/OrTests.cs

```diff
@@ -44,6 +43,69 @@ public void TrueOrGenScalar()
                     ).Value() == true
             );
         }
-    }
-}
 
+        [Fact]
+        public void TestFunc()
+        {
+            Assert.True(
+                    new Or<int>(
+                        input => input > 0,
+                        1, -1, 0
+                    ).Value());
+        }
+
+        [Fact]
+        public void TestFunc2()
+        {
+            Assert.True(
+                    new Or<int>(
+                        input => input > 0,
+                        new List<int>() { 1, -1, 0 }
+                    ).Value());
+        }
+
+        [Fact]
+        public void TestIFunc()
+        {
+            Assert.False(
+                    new Or<int>(
+                        new FuncOf<int, bool>(input => input > 0),
+                        -1, -2, -3
+                    ).Value());
+        }
+
+        [Theory]
+        [InlineData("DB", true)]
+        [InlineData("ABC", true)]
+        [InlineData("DEF", false)]
+        public void TestValueAndFunctionList(string value, bool expected)
```

Comment:
Please choose a Fact about the class for a name

Reply:
Done